src/parsers/perf/perfparser.cpp

```diff
@@ -953,8 +953,17 @@ struct PerfParserPrivate
         auto args = features.cmdline;
         args.removeFirst();
         summaryResult.command = QLatin1String("perf ") + QString::fromUtf8(args.join(' '));
-
-        // TODO: add system info to summary page
+        summaryResult.hostName = QString::fromUtf8(features.hostName);
+        summaryResult.linuxKernelVersion = QString::fromUtf8(features.osRelease);
+        summaryResult.perfVersion = QString::fromUtf8(features.version);
+        summaryResult.cpuDescription = QString::fromUtf8(features.cpuDesc);
+        summaryResult.cpuId = QString::fromUtf8(features.cpuId);
+        summaryResult.cpuArchitecture = QString::fromUtf8(features.arch);
+        summaryResult.cpusOnline = features.nrCpusOnline;
+        summaryResult.cpusAvailable = features.nrCpusAvailable;
+        summaryResult.cpuSiblingCores = QString::fromUtf8(features.siblingCores.join(','));
+        summaryResult.cpuSiblingThreads = QString::fromUtf8(features.siblingThreads.join(','));
```

Comment:
also, looking at the output: could you please sort the sibling threads list before joining it? Currently, I get:

0,4,1,5,2,6,3,7